### PR TITLE
Feature/task app

### PR DESCRIPTION
**Pull request by Silent777**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=738)

## Code reviewers

- [x] @dmytrorevak 
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [x] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [x] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [x]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/task/models.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    status = models.IntegerField(default=0, choices=STATUS_TYPE_CHOICES)
+    created_at = models.DateTimeField(auto_now_add=True, editable=False)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        """
+        Magic method that returns string representation of
+        event instance object.
```
Comment (lhalam) [inline eventually/task/models.py:49]: add empty line

File: eventually/task/models.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+        task = Task()
+        task.title = title
+        task.description = description
+        task.status = status
+
+        try:
+            task.save()
+            return task
+        except IntegrityError:
```
Comment (lhalam) [inline eventually/task/models.py]: Why is IntegrityError, and not ValueError?

File: eventually/task/models.py [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+            return None
```
Comment (lhalam) [inline eventually/task/models.py]: maybе `pass`?

File: eventually/task/models.py [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+        :type task_id: integer
+
+        :return: task object or None if task does not exist
+        """
+
+        try:
+            task = Task.objects.get(id=task_id)
+            return task
+        except Task.DoesNotExist:
+            return None
```
Comment (lhalam) [inline eventually/task/models.py]: maybе `pass`?

**Review by lhalam — CHANGES_REQUESTED** [review]

File: eventually/task/models.py (hunk 3 above)
Comment (ssOleg) [reply eventually/task/models.py]: Or some log...?

File: eventually/task/models.py [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+        """
+
+        task = Task()
+        task.title = title
+        task.description = description
+        task.status = status
+
+        try:
+            task.save()
+            return task
```
Comment (ssOleg) [inline eventually/task/models.py]: Why do you need that `return task`?
return task.save()

File: eventually/task/models.py [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+
+        :param task_id: Unique identificator of task.
+        :type task_id: integer
+
+        :return: task object or None if task does not exist
+        """
+
+        try:
+            task = Task.objects.get(id=task_id)
+            return task
```
Comment (ssOleg) [inline eventually/task/models.py]: Why do you need that `return task`?
return Task.objects.get(id=task_id)

File: eventually/task/models.py (hunk 1 above)
Comment (Silent777) [reply eventually/task/models.py]: +

Comment (Silent777) [reply eventually/task/models.py:49]: added

File: eventually/task/models.py [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+        """
+
+    STATUS_TYPE_CHOICES = (
+        (0, 'ToDo'),
+        (1, 'In Progress'),
+        (2, 'Done')
+    )
+
+    title = models.CharField(max_length=255)
+    description = models.TextField()
```
Comment (redmyt) [inline eventually/task/models.py]: use CharField with 1024 length

File: eventually/task/models.py (hunk 4 above)
Comment (Silent777) [reply eventually/task/models.py]: +pass

File: eventually/task/models.py (hunk 2 above)
Comment (Silent777) [reply eventually/task/models.py]: IntegrityError+ValueError

File: eventually/task/models.py (hunk 3 above)
Comment (Silent777) [reply eventually/task/models.py]: +pass

File: eventually/task/models.py (hunk 6 above)
Comment (Silent777) [reply eventually/task/models.py]: To give the impression that we are doing some work)))
fixed!

**Review by redmyt — DISMISSED** [review]

**Review by lhalam — DISMISSED** [review]

**Comment by lhalam** [comment]
reduce the number of committees.

**Review by redmyt — APPROVED** [review]

**Pull request merged**